tests/zfs-tests/include/libtest.shlib

```diff
@@ -4173,6 +4173,24 @@ function get_arcstat # stat
 	esac
 }
 
+#
+# Wait for the specified arcstat to reach a non-zero plateau.
+#
+function arcstat_plateau # stat
+{
+	typeset stat=$1
+	typeset do_once=true
+
+	while $do_once || [ $stat1 -ne $stat2 ] || [ $stat2 -eq 0 ]; do
```

Comment:
For stats that can go up or down, technically this will find either a plateau or a, err, valley.   Quiescence.  Is that the intent?

Reply:
Essentially inactivity, does not matter if plateau or valley. Quiescence. No activity on L2ARC devices judged by l2_size or complete removal of the L2ARC device as seen by l2_feeds. I will rename it to arcstat_quiescence.